dapp/src/hooks/useStatistics.ts

```diff
@@ -24,12 +24,15 @@ const useStatistics = () => {
     ? 100
     : Math.floor((bitcoinTvl / TOTAL_VALUE_LOCKED_CAP) * 100)
 
+  const remaining = TOTAL_VALUE_LOCKED_CAP - bitcoinTvl
```

Comment:
I think we should not allow `remaining` to have a negative value here.

```suggestion
  const remaining = isCapExceeded ? 0 : TOTAL_VALUE_LOCKED_CAP - bitcoinTvl
```

Reply:
Ref commit: cfae577e